### PR TITLE
Adding random-name

### PR DESCRIPTION
**Pull request by ThomasWollmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ThomasWollmann** [comment]
@conda-forge/python, this PR is review-ready; many thanks if you could have a look!

File: recipes/random-name/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/random_name-{{ version }}.tar.gz
+  sha256: 9905292851b2adceabccafd8998e59acbe3a88e59ac2bd81f8fd9225bd46843b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (ocefpaf) [inline recipes/random-name/meta.yaml:14]: This package declares `scripts` in the setup.py, which would prevent it from being noarch. However, the scripts folder is empty and there aren't any scripts. See https://github.com/phobologic/random_name/blob/0.1.1/setup.py#L31

```suggestion
  number: 0
  noarch: python
```

File: recipes/random-name/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/random-name/meta.yaml:18]:
```suggestion
    - python >=3.6
```

File: recipes/random-name/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/random-name/meta.yaml:21]:
```suggestion
    - python >=3.6
```

**Comment by ThomasWollmann** [comment]
Thanks @ocefpaf for the review. The recipe was generated by `grayskull`

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf for the review. The recipe was generated by `grayskull`

Grayskull did the right thing there. The problem is that the package "lied" to it declaring scripts that do not exist :-)

**Pull request merged**